src/redux/actionTypes/projects.js

```diff
@@ -0,0 +1,18 @@
+const PROJECTS = 'projects';
+
+/**
+ * Save project.
+ */
+const PROJECTS_CREATE = `${PROJECTS}/create`;
+
+/**
+ * Update project.
+ */
+const PROJECTS_UPDATE = `${PROJECTS}/update`;
```

Comment:
how about deleting a project?

Reply:
Deleting and updating a project will be separated into different tickets as they are more involved. Deleting a project requires deleting the related samples and also files. Likewise with updating.

Reply:
ok, makes sense!